tools/pot/openvino/tools/pot/samplers/utils.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2020-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+def format_input_batch(input_batch, idx):
+    if isinstance(input_batch[0], tuple):
```

Comment:
Does this mean that a user can still use the old method and it will continue to work? That would be very nice. If you intend to deprecate this functionality in the future I recommend showing a DeprecationWarning here to allow users time to switch.

Reply:
I agree with the warning if there is a way to show it only once but not for each access to the data.